packages/config/src/__tests__/Config-test.js

```diff
@@ -141,23 +141,25 @@ describe('readConfigJson', () => {
     afterAll(() => vol.reset());
 
     it(`can skip throwing when the app.json is missing`, () => {
-      const { exp, pkg } = readConfigJson('/no-config', true);
+      const { exp, pkg } = readConfigJson('/no-config');
       expect(exp.name).toBe(pkg.name);
       expect(exp.description).toBe(pkg.description);
     });
 
     it(`can skip throwing when the app.json is missing and expo isn't installed`, () => {
-      const { exp, pkg } = readConfigJson('/no-package', true, true);
+      const { exp, pkg } = readConfigJson('/no-package');
       expect(exp.name).toBe(pkg.name);
       expect(exp.description).toBe(pkg.description);
     });
 
     it(`will throw if the app.json is missing`, () => {
-      expect(() => readConfigJson('/no-config')).toThrow(/app.json must include a JSON object./);
+      expect(() => readConfigJson('/no-config', { requireLocalConfig: true })).toThrow(
+        /app.json must include a JSON object./
```

Comment:
I think this error message should include a bit more detail, maybe an example of a valid app.json and/or link to the relevant documentation.